https://bugs.gentoo.org/show_bug.cgi?id=449110

From 9e84e33023a31bc65479784113aad10c69ba0e36 Mon Sep 17 00:00:00 2001
From: Kir Kolyshkin <kir@openvz.org>
Date: Fri, 28 Dec 2012 13:31:07 -0800
Subject: [PATCH] etc/init.d/vz-gentoo: fix missing VZREBOOTDIR

That was pretty lame from my side. Fix for commit af5bf43.

http://bugzilla.openvz.org/2467

Reported-by: Andrei Vinogradov <spamslepnoga@inbox.ru>
Signed-off-by: Kir Kolyshkin <kir@openvz.org>
---
 etc/init.d/vz-gentoo.in |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/etc/init.d/vz-gentoo.in b/etc/init.d/vz-gentoo.in
index 05ef4e7..2dd2186 100755
--- a/etc/init.d/vz-gentoo.in
+++ b/etc/init.d/vz-gentoo.in
@@ -17,6 +17,7 @@ check_config() {
 	: ${VZVEINFO:=/proc/vz/veinfo}
 	: ${VESTAT:=/proc/vz/vestat}
 	: ${VPSCONFDIR:=@VPSCONFDIR@}
+	: ${VZREBOOTDIR:=@VZREBOOTDIR@}
 	: ${VZDEV:=venet0}
 	: ${VZCONF:=@PKGCONFDIR@/vz.conf}
 	: ${VE_STOP_MODE:=suspend}
-- 
1.7.1

